conu/utils/__init__.py

```diff
@@ -123,13 +123,13 @@ def run_cmd(cmd, return_output=False, ignore_status=False, **kwargs):
     """
     logger.debug('command: "%s"' % ' '.join(cmd))
     try:
+        process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
+                                   universal_newlines=True, **kwargs)
+        output = process.communicate()[0]
```

Comment:
I was hoping that communicate would iteratively print output after every newline; unfortunately, it returns all at once.

I guess it's fine for now, we can improve it later.

As a good example here is:
```
run_cmd(["bash", "-c", "for x in `seq 1 10`; do echo $x; sleep 1; done"])
```